src/index.tsx

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 import App from './App';
-
 import './index.css';
+import Root from './routes/root';
 
 i18n

```

Comment:
should be included in `src/location/i18n.ts`.

Then u should import this instance in `App.tsx` instead of `index.tsx`